spec/mailers/notification_mailer_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+RSpec.describe NotificationMailer, type: :mailer do
+  before do
+    workdey_data = Seed.new
+    workdey_data.create_all
+  end
+  describe "send_notifications" do
+    let(:user) { User.first }
+    let(:task) { Task.last }
+    let(:category) { Task.first }
+    let(:taskee) { User.first }
+    let(:tasker) { User.last }
+    let(:nofify) { send_notifications(user, task, category, taskee, tasker) }
+    let(:mail) { NotificationMailer.send_notifications(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.